### ISSUE TITLE
GIP CE: fix remaing FIXME lines

### ISSUE THREAD

**Issue by jouvin** [quattor/template-library-grid#120]
The new GLUE2 compliant GIP CE configuration template (see #110) has a few `FIXME` lines left for minor issues. Would be good to process them at some point...

**Comment by jrha** [comment]
Any chance of this happening for this round?

**Comment by jouvin** [comment]
I don't think so... but in fact I am not aware of any significant issue with the current configuration... My prio for this round is EL7 support in the template library.

**Comment by jrha** [comment]
Ok, I'll bump it and we can check again next time.
